docs/integration/alerts-teams-integration.md

```diff
@@ -0,0 +1,58 @@
+
+To be able to receive incoming notifications we need to add and configure a new incoming webhook. To do that:
+
+**2.** Select **...** options for your team and click **Manage team**:
+
+<img class="content-modal-image" alt="Create Microsoft Teams Integration - Manage Team" src="../../images/integrations/create-teams-integration-manage-team.png" title="Create Microsoft Teams Integration - Manage Team">
+
+**3.** Click the **More Apps** button to display the list of additional applications:
+
+<img class="content-modal-image" alt="Create Microsoft Teams Integration - More Apps" src="../../images/integrations/create-teams-integration-more-apps.png" title="Create Microsoft Teams Integration - More Apps">
+
+**4.** Find the **Incoming Webhook**, chose it and click the **Add to a team** button to include the selected **Incoming Webhook** connector to your team.
+
+<img class="content-modal-image" alt="Create Microsoft Teams Integration - Add App" src="../../images/integrations/create-teams-integration-add-app.png" title="Create Microsoft Teams Integration - Add App">
+
+**5.** Select the channel to which the **Incoming Webhook** should be added and click the **Set up the Connector** button to continue with configuration.
+
+<img class="content-modal-image" alt="Create Microsoft Teams Integration - Select Channel" src="../../images/integrations/create-teams-integration-select-channel.png" title="Create Microsoft Teams Integration - Select Channel">
+
+**6.** Chose the **Configure** button to the right of the **Incoming Webhook** that you just added.
+
+<img class="content-modal-image" alt="Create Microsoft Teams Integration - Configure" src="../../images/integrations/create-teams-integration-configure.png" title="Create Microsoft Teams Integration - Configure">
+
+**7.** Give your webhook a name and click the **Create** button. As an optional step you can also upload a custom icon for your defined webhook. 
+
+<img class="content-modal-image" alt="Create Microsoft Teams Integration - Create Incoming Webhook" src="../../images/integrations/create-teams-integration-create-incoming-webhook.png" title="Create Microsoft Teams Integration - Create Incoming Webhook">
+
+**8.** Finally the webhook URL will be displayed. You need to copy it and store it - it will be used when configuring the Microsoft Teams notification hook in Sematext. Once you have it stored, click **Create** finishing the webhook creation.
+
+<img class="content-modal-image" alt="Create Microsoft Teams Integration - Webhook URL" src="../../images/integrations/create-teams-integration-webhook-url.png" title="Create Microsoft Teams Integration - Webhook URL">
+
+Having the **Incoming Webhook** configured and having the webhook URL we can add the notification to Sematext. 
```

Comment:
Same
(I'm guessing the same in all new docs)

Reply:
Yup, will adjust everywhere. Thanks!